src/analysis/special_functions/gaussian.lean

```diff
@@ -169,3 +169,57 @@ begin
   ... = (sqrt (π / b)) ^ 2 :
     by { rw sq_sqrt, exact div_nonneg pi_pos.le hb.le }
 end
+
+open_locale interval
+
+/- The Gaussian integral on the half-line, `∫ x in Ioi 0, exp (-b * x^2)`. -/
+lemma integral_gaussian_Ioi {b : ℝ} (hb : 0 < b): ∫ x in Ioi 0, exp (-b * x^2) = sqrt (π / b) / 2 :=
```

Comment:
You shouldn't need `hb` here, the formula is always true. You can see in `integral_gaussian` how the other stupid cases are dealt with.